playbooks/roles/streisand-gateway/templates/openssl-local.cnf.j2

```diff
@@ -0,0 +1,85 @@
+distinguished_name = req_distinguished_name
+
+[ ca ]
+default_ca = CA_default
+
+[ CA_default ]
+
+dir = {{ openssl_ca_base }}
+certs = $dir
+crl_dir = $dir
+database = $dir/index.txt
+new_certs_dir = {{ openssl_ca_newcerts_dir }}
+
+certificate = {{ openssl_ca_certificate }}
+serial = $dir/serial
+crl = $dir/crl.pem
+private_key = {{ openssl_ca_key }}
+RANDFILE = $dir/.rand
+
+copy_extensions = copy
+
+x509_extensions = v3_ca
+
+default_days = {{ nginx_days_valid }}
+default_crl_days= 30
+default_md = {{ openssl_ca_default_md }}
+preserve = no
+
+policy = policy_anything_including_no_cn
+
+keyUsage = cRLSign, keyCertSign
+
+subjectKeyIdentifier=hash
+authorityKeyIdentifier=keyid:always,issuer:always
```

Comment:
Ditto what I said for `keyUsage` for the two `subjectKeyIdentifer` and `authorityKeyIdentifier` fields. I believe both are an RFC5280 invention for x509v3

Reply:
Done.